src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -93,14 +95,14 @@ private void handleDeletePerson() {
      */
     @FXML
     private void handleNewPerson() {
-        Optional<Person> newPerson = Optional.of(new Person());
+        Optional<ReadablePerson> prevInput = Optional.of(new Person());
```

Comment:
latestInputPerson? I'm also using it to store the previous input so if the eg. the user tries to add duplicate person we can warn them, then **return them to the dialog and restore their last input**